proto-google-cloud-pubsub-v1/src/main/proto/google/pubsub/v1/pubsub.proto

```diff
@@ -752,6 +752,19 @@ message Subscription {
   // the endpoint will not be made.
   bool detached = 15;
 
+  // If true, Pub/Sub provides the following guarantees for the delivery of
+  // a message with a given value of `message_id` on this subscription:
+  //
+  // * The message sent to a subscriber is guaranteed not to be resent
+  // before the message's acknowledgement deadline expires.
+  // * An acknowledged message will not be resent to a subscriber.
+  //
+  // Note that subscribers may still receive multiple copies of a message
+  // when `enable_exactly_once_delivery` is true if the message was published
+  // multiple times by a publisher client. These copies are  considered distinct
+  // by Pub/Sub and have distinct `message_id` values.
+  bool enable_exactly_once_delivery = 16;
```

Comment:
This should already be there if you sync from master I assume? Can we do that and reduce the diffs?

Reply:
Will do

Reply:
Updated